src/content/docs/codestream/how-use-codestream/performance-monitoring.mdx

```diff
@@ -19,6 +19,66 @@ Go here to get or create your [New Relic user key](https://one.newrelic.com/laun
 
 Once you have your user key, in **Observability** click **Connect to New Relic One**, then paste your user key and click **Connect**.
 
+## Monitor code-level metrics [#code-level]
+
+CodeStream's code-level metrics give you detailed insight into how your code is performing.
+
+![A screenshot of a Python method with average duration, throughput, and error rate telemetry.](./images/code-level-metrics.png "A Python method with average duration, throughput, and error rate metrics.")
+
+<figcaption>When relevant, CodeStream's code-level metrics are shown in a CodeLens above your Python methods.</figcaption>
```

Comment:
It doesn't seem like we define `CodeLens` anywhere in this doc, or link out to it. Could probably just add a link to wherever it is defined!

Reply:
I rewrote this sentence slightly to clarify.